src/drivers/distance_sensor/mb12xx/mb12xx.cpp

```diff
@@ -217,7 +217,11 @@ MB12XX::collect()
 	int ret_val = transfer(nullptr, 0, &val[0], 2);
 
 	if (ret_val < 0) {
-		PX4_ERR("sensor %i read failed, address: 0x%02X", _sensor_index, get_device_address());
+#if defined(__PX4_NUTTX)
```

Comment:
Can you get rid of the platform defines?

Reply:
Did so in change 63cd80a